.pylintrc

```diff
@@ -67,7 +67,11 @@ disable=missing-docstring,
         import-outside-toplevel,
         no-else-continue,
         unnecessary-comprehension,
-        unsubscriptable-object
+        unsubscriptable-object,
+        cyclic-import,
+
+        #TODO: Remove this once todos are done
+        fixme

```

Comment:
how does this work?

Reply:
What this does is globally ignore pylint errors. So the `fixme` line suppresses pylint warnings for the #TODO comments